pantry-finder-api/app/models/zip_agency.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# zip codes the agency services through events
+class ZipAgency < ApplicationRecord
+  self.table_name = 'event_service_geographies'
+
+  alias_attribute :id, :esg_id
+  alias_attribute :zip_code, :geo_value
+
+  has_many :events, primary_key: :event_id, foreign_key: :event_id,
+                    inverse_of: :zip_agencies,
+                    dependent: :restrict_with_exception
+  has_many :agencies, through: :events
+
+  default_scope { geo.active }
+  scope :geo, -> { where(geo_type_id: 1) }
```

Comment:
A more human readable name for this scope might be something like `zip_code_type`, `geo` doesn't really tell me anything about what the scope means.

Reply:
Yeah, scope of "zip_code" would be more in line with the spirit of the data architecture.
```suggestion
  scope :zip_code, -> { where(geo_type_id: 1) }
```

Reply:
agreed